crates/prose-xmpp/src/stanza/vcard4.rs

```diff
@@ -102,6 +110,50 @@ pub struct Org {
     pub value: String,
 }
 
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct UnsupportedProperties(Vec<Element>);
```

Comment:
Since this is used for extensions and unsupported property maybe we could call it more generically a `PropertyContainer`?

Reply:
I tried to use `UnsupportedProperties` as a generic name but I was also thinking about `PropertyContainer`. If I merge `unknown_` and `extensions`, would `unknown` be a good naming choice?

Reply:
- `custom_properties`
- `extra_fields`
- `additional_data`
- `custom_attributes`
- `user_defined_data`

Something like this?

Reply:
It's neither custom nor user-defined since it can include standard properties (like `LANG`). `unknown` seems like a good choice. [RFC 6351 - xCard: vCard XML Representation, section 6](https://datatracker.ietf.org/doc/html/rfc6351#section-6) uses "unrecognized", that might be an option as well.

Reply:
I'd still go with `unknown_properties` then because … `unknown` what?